docs/en/observability/monitor-k8s/monitor-k8s-application-performance.asciidoc

```diff
@@ -0,0 +1,219 @@
+[discrete]
```

Comment:
@eyalkoren -- If you have time, I'd love to get your review on this content as well. Here's a [link](https://observability-docs_151.docs-preview.app.elstc.co/guide/en/observability/master/monitor-kubernetes.html#monitor-kubernetes-application-performance) to this content in the preview build.

Reply:
Of course, there's little I wouldn't I do if you ask.